docs/json/swagger.json

```diff
@@ -118,12 +118,12 @@
         }, {
           "name" : "prefixes",
           "in" : "query",
-          "description" : "JSON format prefix map, used to expand prefixes in the 'object' expression",
+          "description" : "JSON format prefix map, used to expand prefixes in the 'object' expression.  A JSON format prefix map is a JSON object whose keys are prefixes without the colon separator (e.g., \"owl\"), and whose values are IRI prefixes (typically WITH the hash-mark, \"#\" suffix).  Because these are passed as parameters, they need to be URL-encoded.  An example JSON-format prefix map would be: \'{\"obo\": \"http://purl.obolibrary.org/obo/\", \"part_of\": \"http://purl.obolibrary.org/obo/BFO_0000050\"}\'"
           "required" : false,
           "schema" : {
             "type" : "string",
             "format" : "JSON",
-            "example" : "{\"obo\": \"http://purl.obolibrary.org/obo/\", \"part_of\": \"http://purl.obolibrary.org/obo/BFO_0000050\"}"
+            "example" : "%7B%22obo%22%3A%20%22http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2F%22%2C%20%22part_of%22%3A%20%22http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050%22%7D"
```

Comment:
I don't think it's correct to URL-encode the example value. Any value would need to be URL-encoded, and if you use the Swagger UI, it will do that for you.

Reply:
I was using `curl` and Postman, and both of them made me do the URL encoding myself.  For `curl` there is the convenience argument `--data-urlencode` and Postman has a UI widget to do this.  But in both cases it was down to me to understand that the URL-encoding was required.  I think anyone who is working programmatically would have this problem (I don't believe that the openapi-generator would be smart enough to automate this).

BTW, I couldn't figure out how to use the UI.  When I started up Owlery, I could not figure out how to get access to the Swagger page.  I could get access to it from the Owlery GitHub repo, but that, of course, was not connected to _my_ Owlery server.